standards/out-of-hours.md

```diff
@@ -0,0 +1,62 @@
+
+## Managing a sustainable out of hours rota
+
+Being on call can be stressful, and can impact people's mental health and personal lives if not managed well. If that happens, people will withdraw from the rota, leaving the service unsupported.
+
+To avoid that, a few core criteria must be met:
+
+- No one person is on call 50% of the time or more, and ideally no more than 30% of the time. This, along with the next criteria, mean that you will need at least six people, and ideally eight, to run a rota sustainably.
+- At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).
+- If your service regularly has problems out of hours, or problems typically take multiple hours to resolve, you will need to take that into account when designing your rota. These sorts of situations will require more people, shorter shifts, or more layers of support.
+- If your service regularly has problems out of hours, your incident retrospectives should be ensuring that appropriate learning and investment is happening to improve how often the service has problems.
+- The rota repeats regularly, so people know when they are on call over the coming weeks and months, and can plan accordingly.
+- People on call are appropriately paid for being available on call and for responding to incidents.
+- People on call know when they are expected to be available when on call, whether 24/7, 7am-10pm, or some other, agreed hours.
+- People on call can trade shifts with one another when they need to.
+- People on call for the first time are on call with someone experienced to being on call on their support rota, and ideally someone they know well.
+- People on call understand the circumstances under which they will be paged. This should ideally be a clearly defined list of alerts with agreed steps to resolve, but some services may not yet be in a position to support this.
+
+### Contacting people on call about an issue
+
+To allow people to manage how they are contacted when on call, the MOJ uses [PagerDuty](https://www.pagerduty.com). PagerDuty (and systems like it) allows people on call to specify how they prefer to be contacted (in-app notifications, phone calls, text messages) and when, relative to an incident being raised. It also ensures that the people on call are paged according to pre-defined escalation policies (primary first, then secondary, then repeat).
+
+People on call should *never* be contacted directly to report issues, especially not through their personal phone number. Contacting individuals directly creates a dependency on them, making them implicitly on call 100% of the time. This is not good for individuals' mental health, and can result in them withdrawing entirely from support or even leaving the organisation, meaning we entirely lose their knowledge and skill at supporting the system, rather than allowing others to learn from it and them to work at a sustainable level.
```

Comment:
Full marks for this paragraph! A lot of organisations don't do this properly, or at all. Calling it out clearly like this is really good.